### ISSUE TITLE
Feed problems when it is empty(UI)

### ISSUE THREAD

**Issue by bastiao** [bioinformatics-ua/emif-fb#905]
When you do not have feed, we need to improve the ui of the popup: "Do you know how subscribe databases works ?"

Also it raises the following exception: 
```
Uncaught ReferenceError: hasDatatable is not defined
widget.userstats.js:48 Object
```

Please, fix agains the master branch.

**Comment by ribeiroct** [comment]
This was partially fixed today on master (here #911  )

**Comment by ribeiroct** [comment]
Fixed popup on the same pull request.
